### ISSUE TITLE
Multi-Book years

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#23]
Clarify that jumping on the book name in multi-book years is still a foul, no different from single-book years. 

Also clarify that when a foul is awarded when a quizzer jumps before the chapter number (on Q and Ref) or before the QM discernibly begins to read the question (all other question types), the question is NOT to be thrown out and replaced.

Of course, since it's v-next we can decide if someone wants to propose something different. (Scott does not).
In fact, I'd propose that any jump after a QM discernibly begins to read the question should be an awarded jump, even on Q/Ref. Greatly simplifies things.

**Comment by ARMediting** [comment]
Two thoughts:

Why is there a need to clarify based on the different material years? Currently in 4.5., all fouls are given before the chapter number, not the book. (I'm fine with this, by the way; the only time it would become important is if we reinstated Philemon or added another similarly short book.)

Also, what would happen if a QM notices a quizzer jump before the chapter number or discernibly beginning but isn't able to stop reading? Would that just always be a bleed, or would that require throwing out the question?

**Comment by jswingle** [comment]
I think this doesn't need to be clarified. Any QM not granting a foul, citing non-rulebook reasons like "but they jumped on the book name and there are multiple books this year" needs to study the rulebook better anyways, this is already super clear and non-debatable. (And can be checked by the coaches' ability to protest if the QM fails to follow the rules.)

I would give a +1 to the idea of just awarding the jump anyways, I can't see a downside to that.

**Comment by kclimenhaga** [comment]
Generally, when I see quizzers foul on reference or quote questions, it's not internationals-level quizzers; it's newer quizzers who got excited because no one was jumping and didn't think through the fact that they don't have enough information to answer yet. I wouldn't want to make those quizzers get up and give an answer, so I prefer calling that a foul.

**Comment by josiah-leinbach** [comment]
Agreed with Jeremy that this is already covered in the rules, and agreed with Kristen on the circumstances behind most fouls in these cases and why we ought keep the rule as is.

**Comment by jswingle** [comment]
I would add to Kristen's reasoning that I think there's also a point to fouling quizzers on early Reference jumps at higher levels of competition as well: quizzers accidentally doing quick prejumps and forgetting that it's a Reference question. It's not an extremely common occurrence, but I think it's kind of an unfortunate way to burn Reference questions in a quiz. 

I'm all in favor of changing so that the point at which the _name of the book is said_ is when the jump is awarded instead of a foul. I haven't done the math on it, but I would guess there aren't too many MACRs in, say, 2 Peter. So there could possibly be important strategic information from the book name alone.
